estree/index.d.ts

```diff
@@ -488,7 +488,8 @@ interface BaseModuleSpecifier extends BaseNode {
 
 export interface ImportDeclaration extends BaseModuleDeclaration {
   type: "ImportDeclaration";
-  specifiers: (ImportSpecifier|ImportDefaultSpecifier|ImportNamespaceSpecifier)[];
+  specifiers: Array<
+      ImportSpecifier | ImportDefaultSpecifier | ImportNamespaceSpecifier>;
```

Comment:
Can you please not break the line here? IMO it would look cleaner on the same one.